functions/models/src/helpers/schemaConverter.ts

```diff
@@ -23,9 +23,17 @@ export class SchemaConverter<Schema extends z.ZodType<any, any, any>> {
 
   constructor(input: {
     schema: Schema
-    encode: (value: z.output<Schema>) => z.input<Schema>
+    encode: (value: z.output<Schema>) => Encoded
   }) {
     this.schema = input.schema
     this.encode = input.encode
   }
 }
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type InferEncoded<Input> =
```

Comment:
This is just a helper to get `Encoded` from schemas. Example: `type MedicationClass = InferEncoded<typeof medicationClassConverter>`